src/Files.App/Data/Items/PathBreadcrumbItem.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace Files.App.Data.Items
 {
-	public class PathBoxItem
+	public class PathBreadcrumbItem
 	{
-		public string? Title { get; set; }
+		public string? Name { get; set; }
```

Comment:
How about "FolderName"?

Reply:
We may use somewhere else (eg. nested settings page) and who knows it, so I’d like to keep this general. But I’m fine if you are ok with FolderName.

Reply:
ItemName

Reply:
I'm not sure if we'll use this anywhere else, but we can keep it generic just in case.